javascript/durian.js-fastify/config.yaml

```diff
@@ -5,3 +5,5 @@ framework:
     - bun
   bootstrap:
     - bun install
+  command:
+    - bun run pm2
```

Comment:
This should be a string, not an array.

Also this should be set in javascript/config.yaml file